### DISCUSSION TITLE
Minimum Python Version

### DISCUSSION THREAD

**Discussion by WesleyJohnson** [saleweaver/python-amazon-sp-api#573] [Q&A]
I am sure I am overlooking something obvious, but what is the minimum python version needed to run this? I was trying to run on 3.5, but f-strings weren't introduced until 3.6 which sp-api uses.

Thanks!

**Comment by gmweinberg** [comment]
It has been working for me with 3.6, so that must be the minimum if it doesn't work with 3.5.
